SpecRunner.html

```diff
@@ -4,7 +4,7 @@
     <title>Recursion Test Suite</title>
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
     <link rel="stylesheet" href="lib/css/mocha.css" />
-    <script src="lib/jquery.js"></script>
+    <!-- <script src="lib/jquery.js"></script> -->
```

Comment:
Can this be deleted?

Reply:
yes, probably. I can't tell that it's actually being used by anything, but I'm hesitant to remove this line or the module itself.